### PR TITLE
Refs [TASK][36219] Create base API and BindingData

### PR DESCRIPTION
**Pull request by vvanan158**
## Related Tickets
- [#Task 36219: Create base API and BindingData](https://edu-redmine.sun-asterisk.vn/issues/36219)
## WHAT
-  setup base Api and databinding 
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/unsplash_01/extensions/ViewExt.kt [hunk 1]
```diff
@@ -20,3 +21,12 @@ fun ImageView.loadFromUrl(url: String) {
         .skipMemoryCache(false)
         .into(this)
 }
+
+fun ImageView.loadUrlStaggered(url: String) {
```
Comment (datdt-1935) [inline app/src/main/java/com/sun/unsplash_01/extensions/ViewExt.kt]: Để ra ImageView extension

File: app/src/main/java/com/sun/unsplash_01/extensions/plusAssign.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.sun.unsplash_01.extensions
+
+import androidx.lifecycle.MutableLiveData
+
```
Comment (datdt-1935) [inline app/src/main/java/com/sun/unsplash_01/extensions/plusAssign.kt]: đôit tên file thì MutableLiveData extension

File: app/src/main/java/com/sun/unsplash_01/binding/loadImage.kt [hunk 3]
```diff
@@ -0,0 +1,70 @@
+package com.sun.unsplash_01.binding
+
+import android.widget.ImageView
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/unsplash_01/binding/loadImage.kt]: đổi tên file thành BindingAdapter

File: app/src/main/java/com/sun/unsplash_01/extensions/MutableLiveData.kt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package com.sun.unsplash_01.extensions
+
+import androidx.lifecycle.MutableLiveData
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/unsplash_01/extensions/MutableLiveData.kt]: đổi tên file thành MutableLiveDataExt

**Review by datdt-1935 — APPROVED** [review]

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**